code/obj/item/paper.dm

```diff
@@ -1120,6 +1120,25 @@ Only trained personnel should operate station systems. Follow all procedures car
 	return
 */ //TODO: FIX
 
+/obj/item/paper_bin/robot
+	name = "semi-automatic paper bin"
+	var/next_generate = 0
+
+	attack_self(mob/user as mob)
+		if (src.amount < 1 && isnull(locate(/obj/item/paper) in src))
+			if (src.next_generate < ticker.round_elapsed_ticks && prob(25))
```

Comment:
Unsure why you want `prob(25)` in here.

Reply:
To expand on this: if this is empty and someone attempts to use it (even if ready to produce another one), they have a 75% chance of being told "Nothing left in...". There is no indication that it will produce more if you try it again.

Probably want some feedback about how it's still "recharging" or whatever. `prob(25)` is just a frustration thing here that doesn't add anything - they already have to wait 5 seconds, why make them have to spam click the button?